### PR TITLE
Cynthiar/remove init

### PR DESCRIPTION
**Pull request by cynthia-r**

### CODE REVIEW

File: PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1 [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This script illustrate how to manage extended properties on entities
```
Comment (lksoe) [inline PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1]: -> illustrates

File: PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1 [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Extended properties can be added to various entities. Examples include 'Customer', 'CustomerDevice', 'CustomerEndUser'.
```
Comment (lksoe) [inline PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1]: -> , and 'CustomerEndUser'.

File: PowerShell/BuildingBlocks/CustomerAndResources/AssignSubscriptions.ps1 [hunk 3]
```diff
@@ -1,23 +1,33 @@
-﻿# Initialize the context
-.\Init.ps1 -Customer
+﻿# Authenticate
```
Comment (AlexTkatch) [inline PowerShell/BuildingBlocks/CustomerAndResources/AssignSubscriptions.ps1]: Consider adding a short comment about what the script does here and in other scripts, similar to ManageExtendedProperties.ps1.

File: PowerShell/BuildingBlocks/SDKOperations/RetrieveEntities.ps1 [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
+# Retrieve a customer and get a ticket for it
+$customerId = [GUID](Read-Host -Prompt 'Customer ID') 
+$customer = Get-BT_Customer -Ticket $ticket -Id $customerId
+$customerTicket = Get-BT_Ticket -Ticket $ticket -OrganizationId $customer.OrganizationId
 
 # Here are 3 common ways to retrieve entities
 # 1. Retrieve all the endpoints under a customer and process them one by one
 # Use -RetrieveAll and piping with ForEach 
-Get-BT_Endpoint -Ticket $mspc.CustomerTicket -RetrieveAll | ForEach {
+Get-BT_Endpoint -Ticket $customerTicket -RetrieveAll | ForEach {
```
Comment (AlexTkatch) [inline PowerShell/BuildingBlocks/SDKOperations/RetrieveEntities.ps1]: Q: by default I believe PowerShell does not filter out soft-deleted entities. If the caller needs to only get non-deleted records, they need to manually pass "-IsDeleted $false". Does it makes sense to include this in the example?

File: PowerShell/BuildingBlocks/ServicesAndTasks/DeployRunbook.ps1 [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿# Initialize variables
+$offeringMetadataId = [GUID](Read-Host -Prompt 'Runbook ID')
+$customerId = [GUID](Read-Host -Prompt 'Customer ID')    
+
+# Authenticate
```
Comment (AlexTkatch) [inline PowerShell/BuildingBlocks/ServicesAndTasks/DeployRunbook.ps1]: In other scripts, credentials are requested first, then customer id. Could be useful to normalize here as well.

File: PowerShell/BuildingBlocks/ServicesAndTasks/ExportRunbook.ps1 [hunk 6]
```diff
@@ -1,12 +1,10 @@
-param(
-    [Parameter(Mandatory=$true)]
-    [guid]$OfferingMetadataId,
+# Initialize variables
+$offeringMetadataId = [GUID](Read-Host -Prompt 'Runbook ID')
+$folderPath = Read-Host -Prompt 'Folder path to export to'
 
-    [Parameter(Mandatory=$true)]
-    [string]$FolderPath)
+# Authenticate
```
Comment (AlexTkatch) [inline PowerShell/BuildingBlocks/ServicesAndTasks/ExportRunbook.ps1]: Same question about the order in which information is requested here.

File: PowerShell/BuildingBlocks/CustomerAndResources/AssignSubscriptions.ps1 (hunk 3 above)
Comment (cynthia-r) [reply PowerShell/BuildingBlocks/CustomerAndResources/AssignSubscriptions.ps1]: Done

File: PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1 (hunk 1 above)
Comment (cynthia-r) [reply PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1]: Fixed

File: PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1 (hunk 2 above)
Comment (cynthia-r) [reply PowerShell/BuildingBlocks/CustomerAndResources/ManageExtendedProperties.ps1]: Fixed

File: PowerShell/BuildingBlocks/SDKOperations/RetrieveEntities.ps1 (hunk 4 above)
Comment (cynthia-r) [reply PowerShell/BuildingBlocks/SDKOperations/RetrieveEntities.ps1]: Makes sense, added

File: PowerShell/BuildingBlocks/ServicesAndTasks/DeployRunbook.ps1 (hunk 5 above)
Comment (cynthia-r) [reply PowerShell/BuildingBlocks/ServicesAndTasks/DeployRunbook.ps1]: Good catch, fixed

File: PowerShell/BuildingBlocks/ServicesAndTasks/ExportRunbook.ps1 (hunk 6 above)
Comment (cynthia-r) [reply PowerShell/BuildingBlocks/ServicesAndTasks/ExportRunbook.ps1]: Fixed

**Pull request merged**